### ISSUE TITLE
samtoh5 not found

### ISSUE THREAD

**Issue by pawelosipowski** [PacificBiosciences/pitchfork#297]
Hi, when running blasr I teg error of `samtoh5 not found`. What is it?

**Comment by mhsieh** [comment]
On Oct 1, 2016, at 00:29, Pawel <notifications@github.com<mailto:notifications@github.com>> wrote:
Hi, when running blasr I teg error of samtoh5 not found. What is it?
If possible, please ask blasr question in blasr issue tracking. I'm going to look into it, if pitchfork can't directly help you find the solution, i will close this issue.
Best, -mj

**Comment by mhsieh** [comment]
If possible, please ask blasr question in blasr issue tracking. I'm going to look into it, if pitchfork can't directly help you find the solution, i will close this issue.

**Comment by pawelosipowski** [comment]
Ok. Thanks!

**Comment by mhsieh** [comment]
I now remember why I didn't add the binaries to the list, those executables are not in current 3.x distribution, but we have not delete them from the source code. I am adding them back until the next dead code clean-up, let's hope you guys can move on to use 3.x software before they are gone.

TL;DR - next time you use the latest pitchfork to build blasr, you will get those, for now you can just copy them from workspace/blasr/utils/

**Comment by mhsieh** [comment]
confirmed locally
